src/vs/workbench/parts/terminal/electron-browser/terminalLinkHandler.ts

```diff
@@ -39,12 +42,42 @@ export class TerminalLinkHandler {
 	private _tooltipDisposables: IDisposable[] = [];
 	private _widgetManager: TerminalWidgetManager;
 
+	// Changing any regex may effect this value, hence changes this as well if required.
+	private _winLineAndColumnMatchIndex = 12;
+	private _unixLineAndColumnMatchIndex = 15;
+
+	private _lineAndColumnClauses = [
```

Comment:
Any reason this isn't a constant up the top so it's shared between multiple instances of `TerminalLinkHandler`?